.circleci/config.yml

```diff
@@ -146,7 +146,7 @@ jobs:
       - run:
           command: git status
       - run: |
-          if ! git diff --exit-code docs/gen > /dev/null; then
+          if ! git diff --exit-code embedJson/gen > /dev/null; then
```

Comment:
Why did we move all the documentation generation out of `docs/gen` and into `embedJson/gen`?